bots/tools/self_tools.py

```
@@ -660,3 +599,266 @@ def _remove_dummy_from_tree(node, dummy_content):
+        # Count successful subagents
+        success_count = sum(1 for r in responses if r and not r.startswith("Error"))
+
+        # Format results
+        result_parts = [f"Successfully completed {success_count}/{len(task_list)} subagent tasks.\n"]
+        for i, (task, response) in enumerate(zip(task_list, responses), 1):
+            result_parts.append(f"\n--- Subagent {i}: {task[:50]}{'...' if len(task) > 50 else ''} ---")
+            result_parts.append(response)
+
+        return "\n".join(result_parts)
+
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)